Python/Product/Analysis/VariableDef.cs

```diff
@@ -85,10 +87,23 @@ abstract class DependentData<TStorageType> where TStorageType : DependencyInfo {
 
         public bool AddDependency(AnalysisUnit unit) {
             if (!unit.ForEval) {
-                return GetDependentItems(unit.DeclaringModule.ProjectEntry).AddDependentUnit(unit);
+                return GetDependentItems(unit.DependencyProject).AddDependentUnit(unit);
             }
             return false;
         }
+
+        protected static bool IsVisible(IVersioned accessor, IVersioned declaringScope, IVersioned assigningScope) {
+            return true;
+            /*
+            if (accessor != null && accessor.IsVisible(assigningScope)) {
+                return true;
+            }
+            if (declaringScope != null && declaringScope.IsVisible(assigningScope)) {
+                return true;
+            }
+            return false;*/
```

Comment:
This is actually called...  We need to figure out what policy we want to enforce here.   This is what NTVS did, but Python will be different.  I'm leaving this here as food for thought

---
In reply to: [63613180](https://github.com/Microsoft/PTVS/pull/1299#discussion_r63613180) [](ancestors = 63613180)